ecommerce/api.py

```diff
@@ -522,6 +621,32 @@ def get_product_version_price_with_discount(*, coupon_version, product_version):
     return positive_or_zero(price - discount_amount)
 
 
+def get_product_version_price_with_discount_tax(
+    *, coupon_version, product_version, tax_rate
+):
+    """
+    Uses get_product_version_price_with_discount to get the price for the item
+    and then add the specified tax amount to it.
+
+    Args:
+        coupon_version (CouponVersion): the CouponVersion object
+        product_version (ProductVersion): the ProductVersion object
+        tax_rate (Decimal): the tax rate to apply
+
+    Returns:
+        tuple: product price and tax assessed
+    """
+
+    product_version_price = get_product_version_price_with_discount(
+        coupon_version=coupon_version, product_version=product_version
+    )
+
+    return (
+        product_version_price,
+        (0 if not tax_rate else decimal.Decimal() * (tax_rate / 100)),
```

Comment:
I think you forgot to do 
```suggestion
        (0 if not tax_rate else decimal.Decimal(product_version_price) * (tax_rate / 100)),
```

Reply:
Resolved in https://github.com/mitodl/mitxpro/pull/2772/commits/81a01c16dac5914a99144b5647cd65e612bd483a (or thereabouts)